crates/milli/src/update/new/channel.rs

```diff
@@ -17,8 +17,8 @@ use crate::vector::Embedding;
 use crate::{DocumentId, Index};
 
 /// The capacity of the channel is currently in number of messages.
-pub fn extractor_writer_channel(cap: usize) -> (ExtractorSender, WriterReceiver) {
-    let (sender, receiver) = crossbeam_channel::bounded(cap);
+pub fn extractor_writer_channel(_cap: usize) -> (ExtractorSender, WriterReceiver) {
+    let (sender, receiver) = crossbeam_channel::unbounded();
```

Comment:
Can we rollback this, please? I am touching this part in my PR with BBQueues and it removes the memory control we were working on.

Reply:
removed